integrations/facebook-pixel/test/index.test.js

```diff
@@ -1992,6 +2107,25 @@ describe('Facebook Pixel', function() {
 
         analytics.called(window.fbq, 'trackSingle', options.pixelId, '123456');
       });
+
+      it('should send facebook pixel properties for product search', function() {
+        analytics.track('Products Searched', {
+          product_id: '507f1f77bcf86cd799439011',
+          query: 'yo',
+          currency: 'USD',
+          category: 'Cars',
+          revenue: 30
+        });
+        analytics.called(window.fbq, 'trackSingle', options.pixelId, 'Search', {
+          search_string: 'yo',
+          content_category: 'Cars',
+          content_ids: ['507f1f77bcf86cd799439011'],
+          contents: [],
```

Comment:
Why is the contents array empty?

Reply:
Ohh, i missed this, i was under impression that contents will purely derived from products array.

Reply:
@Brennan removing this test case completely as there is no corresponding specs present here: https://segment.com/docs/spec/ecommerce/v2/#products-searched for missing properties.